### ISSUE TITLE
Can be worked around using Lambda Menu

### ISSUE THREAD

**Issue by davwheat** [TomGrobbe/NoPlayerBlips#5]
You can use `Show FOV Arcs` (or similar) option in lambda menu to see where people are.

**Comment by TomGrobbe** [comment]
show fov arcs?

**Comment by davwheat** [comment]
Yes. On the Settings menu, then Police FOV Cones. This enables the cones on
ALL PLAYERS And you can see their location using that.
